analyzer/src/main/kotlin/curation/OrtConfigPackageCurationProvider.kt

```diff
@@ -61,7 +59,7 @@ open class OrtConfigPackageCurationProvider : PackageCurationProvider {
         val file = curationsDir.resolve("curations").resolve(pkgId.toCurationPath())
         return if (file.isFile) {
             runCatching {
-                yamlMapper.readValue<List<PackageCuration>>(file).filter { it.isApplicable(pkgId) }
+                file.readValueOrDefault(emptyList<PackageCuration>()).filter { it.isApplicable(pkgId) }
```

Comment:
Please provide a rationale why tolerating empty files is better.
The argument of consistency would also allow aligning the other way round.

Reply:
> The argument of consistency would also allow aligning the other way round.

Well, given that you guys are for failing hard, it indeed makes more sense to align the other way around.